rbtcwrapperproxy/contracts/RBTCWrapperProxy.sol

```diff
@@ -251,13 +289,15 @@ contract RBTCWrapperProxy is ContractRegistryClient {
      * 4.Sneds RBTC to user
```

Comment:
typo - `sends`
you also might want to update process description

same for `removeLiquidityFromV1`